networking_cisco/plugins/cisco/db/l3/l3_router_appliance_db.py

```diff
@@ -451,20 +503,41 @@ def remove_router_interface(self, context, router_id, interface_info):
         info = super(L3RouterApplianceDBMixin, self).remove_router_interface(
             context, router_id, interface_info)
         self.notify_router_interface_action(context, info, routers, 'remove')
+        if driver:
+            driver.remove_router_interface_postcommit(context, port_ctxt)
         return info
 
     def create_floatingip(self, context, floatingip,
                           initial_status=FLOATINGIP_STATUS_ACTIVE):
         info = super(L3RouterApplianceDBMixin, self).create_floatingip(
```

Comment:
Yeah -- there's no router context, so we can't get a router type driver to invoke a precommit. It's worth noting that a subsequent patch that adds support for GBP workflow does add a call to this API, but the router type selected will default to ACI so that the precommit call can be made.